e2e/autoscript-suites/src/suites/config-request-middleware.test.ts

```diff
@@ -44,23 +44,6 @@ test.describe('Test request middleware with login flow', () => {
     expect(messageArray.includes('User info successfully responded')).toBe(true);
     expect(messageArray.includes('Logout successful')).toBe(true);
 
-    // Test for absence of error logs for FRUser.logout
-    expect(messageArray.includes('Session logout was not successful')).toBe(false);
-    expect(messageArray.includes('OAuth endSession was not successful')).toBe(false);
-    expect(messageArray.includes('OAuth revokeToken was not successful')).toBe(false);
```

Comment:
Why were these lines removed?

Reply:
hmm looks like i got greedy let me double check.

Reply:
looks like just the diff was moved?

Reply:
Oh, okay, I see it. Looks good.